docs/concepts/resource-manager.md

```diff
@@ -943,7 +943,7 @@ The webhook performs the following actions:
       In case the number of replicas is larger than twice the number of zones, then the `maxSkew=2` for the second spread constraints.
       The `minDomains` calculation is based on whatever value is lower - (maximum) replicas or number of zones. This is the number of minimum domains required to schedule pods in a highly available manner.
 
-   Independent on the number of zones, when the `high-availability-config.resources.gardener.cloud/failure-tolerance-type` annotation is set and NOT empty, then the `whenUnsatisfiable` is set to `DoNotSchedule` for the constraint with `topologyKey=kubernetes.io/hostname` (which enforces the node-spread).
+   Independent on the number of zones, when the `high-availability-config.resources.gardener.cloud/host-spread` annotation is set to `true` or the `high-availability-config.resources.gardener.cloud/failure-tolerance-type` annotation is set and NOT empty, then the `whenUnsatisfiable` is set to `DoNotSchedule` for the constraint with `topologyKey=kubernetes.io/hostname` (which enforces the node-spread).
```

Comment:
Just a nit: This sentence is really long now and not easy to understand. Maybe, it would make sense to split it up.

Reply:
Addressed with https://github.com/gardener/gardener/pull/8970/commits/cb5b95fa1d184a09ae82c166289b4eb1339e5884.